app/serializers/user_serializer.rb

```diff
@@ -3,7 +3,7 @@ class UserSerializer
   include RecentLinkSerializer
   include MediaLinksSerializer
 
-  attributes :id, :display_name, :credited_name, :email, :created_at,
+  attributes :id, :login, :display_name, :credited_name, :email, :created_at,
     :updated_at, :type, :firebase_auth_token, :global_email_communication,
     :project_email_communication, :beta_email_communication,
     :slug, :max_subjects, :uploaded_subjects_count, :admin
```

Comment:
There's a problem with the identity group build:

``` ruby
# User#build_identity_group
identity_membership.build_user_group(display_name: display_name, name: login)

# UserGroup
validates :display_name, format: { without: /\$|@|\s+/ }, unless: :identity?

# UserGroup#identity?
!!memberships.where(identity: true).pluck(:identity).first
```

`identity?` will never be true on creation since that query can't run on the unpersisted records.
Since User#display_name can be unrestricted, shouldn't UserGroup#display_name be unrestricted?